mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/JPAMailboxFixture.java

```diff
@@ -35,7 +32,8 @@
         JPAProperty.class,
         JPAUserFlag.class,
         JPAMailboxAnnotation.class,
-        JPASubscription.class};
+        JPASubscription.class,
+        JPAMailboxAnnotationId.class};
```

Comment:
Then why don't you add MailboxIdUidKey.class if you added JPAMailboxAnnotationId.class ?